__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const devData = require("../db/data/development-data/index");
+const runSeed = require("../db/seeds/run-seed");
+
+describe("app", () => {
+	describe("/api/categories", () => {
+		it("should respond with an array of categories", () => {
+			return request(app)
+				.get("/api/categories")
+				.expect(200)
+				.then((response) => {
+					const { body } = response;
+					const categories = body;
+					expect(Array.isArray(categories)).toBe(true);
+					expect(categories[0]).toMatchObject({
+						slug: expect.any(String),
+						description: expect.any(String),
+					});
```

Comment:
This only checks the first element, consider using iteration to check every element

Reply:
Done!